.cspell.json

```diff
@@ -109,15 +103,14 @@
     "vstack",
     "walkthroughs",
     "WCAG",
-    "webfont",
-    "Woohoo",
     "zindex",
     "بالعالم",
     "مرحبًا"
   ],
-  "language": "en,en-US",
+  "language": "en, en-US",
```

Comment:
I wonder, if we just go with `en-US`, will that mean that only US English will be enforced? We already follow US English, so this might be a good change.